neo.UnitTests/UT_NeoToken.cs

```diff
@@ -124,6 +148,80 @@ public void Check_SupportedStandards()
                 .Should().BeEquivalentTo(new string[] { "NEP-5", "NEP-10" });
         }
 
+        [TestMethod]
+        public void Check_UnclaimedGas()
+        {
+            var snapshot = Store.GetSnapshot().Clone();
+            snapshot.PersistingBlock = new Block() { Index = 1000 };
+
+            byte[] from = Contract.CreateMultiSigRedeemScript(Blockchain.StandbyValidators.Length / 2 + 1,
+                Blockchain.StandbyValidators).ToScriptHash().ToArray();
+
+            Check_Initialize(snapshot);
+
+            var unclaim = Check_UnclaimedGas(snapshot, from);
+            unclaim.Item1.Should().Be(new BigInteger(800000000000));
+            unclaim.Item2.Should().BeTrue();
+
+            unclaim = Check_UnclaimedGas(snapshot, new byte[19]);
+            unclaim.Item1.Should().Be(BigInteger.Zero);
+            unclaim.Item2.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public void Check_Transfer()
+        {
+            var snapshot = Store.GetSnapshot().Clone();
+            snapshot.PersistingBlock = new Block() { Index = 1000 };
+
+            byte[] from = Contract.CreateMultiSigRedeemScript(Blockchain.StandbyValidators.Length / 2 + 1,
+                Blockchain.StandbyValidators).ToScriptHash().ToArray();
+
+            byte[] to = new byte[20];
+
+            Check_Initialize(snapshot);
+
+            var keyCount = snapshot.Storages.GetChangeSet().Count();
+
+            // Check unclaim
+
+            var unclaim = Check_UnclaimedGas(snapshot, from);
+            unclaim.Item1.Should().Be(new BigInteger(800000000000));
+            unclaim.Item2.Should().BeTrue();
+
+            // Transfer
+
+            Check_Transfer(snapshot, from, to, BigInteger.One, true).Should().BeTrue();
```

Comment:
`Check_Transfer(snapshot, from, to, BigInteger.Zero, true).Should().BeTrue();`

If you pass BigInteger.Zero it will should return true, right?
Is this expected?

Reply:
https://github.com/neo-project/neo/pull/717#discussion_r281221909